tests/fail/both_borrows/aliasing_mut4.tree.stderr

```diff
@@ -6,8 +6,8 @@ LL |         ptr::write(dest, src);
    |
    = help: this indicates a potential bug in the program: it performed an invalid operation, but the Tree Borrows rules it violated are still experimental
    = help: the accessed tag <TAG> is foreign to the protected tag <TAG> (i.e., it is not a child)
-   = help: this foreign write access would cause the protected tag <TAG> to transition from Frozen to Disabled
-   = help: this transition would be a loss of read permissions, which is not allowed for protected tags
+   = help: this foreign write access would cause the protected tag <TAG> currently Frozen to become Disabled
```

Comment:
```suggestion
   = help: this foreign write access would cause the protected tag <TAG> (currently Frozen) to become Disabled
```
this reads a bit strange without the parentheses